tasks/job.yml

```diff
@@ -21,4 +21,5 @@
     time_zone: "{{ cohesity_protection.time_zone | default('America/Los_Angeles') }}"
     start_time: "{{ cohesity_protection.start_time | default('') }}"
     delete_sources:  "{{ cohesity_protection.delete_sources | default(False) }}"
+    disable_indexing: "{{ cohesity_protection.disable_indexing | default(True) }}"
```

Comment:
Check if this is applicable to vmware /physical/nas/oracle .. the supported configs..else we may have to : 
1. Enhance documentation 
2. Validations in code to catch any other env. is given for indexing